charts/flink-job/Chart.yaml

```
@@ -7,7 +7,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.3.8
+version: 0.3.9
```

This feels like a minor and not patch.
```suggestion
version: 0.4.0
```